Modules/Apps/NirSoft/NirSoft_WebBrowserPassView.mkape

```diff
@@ -0,0 +1,24 @@
+Description: 'WebBrowserPassView - Nirsoft'
+Category: BrowsingPass
+Author: Pedro Sanchez Cordero (conexioninversa)
+Version: 1.0
+Id: a216d646-cbac-45bb-8340-e0168baac683
+BinaryUrl: https://www.nirsoft.net/toolsdownload/webbrowserpassview.zip
+ExportFormat: csv
+Processors:
+    -
+        Executable: WebBrowserPassView.exe
+        CommandLine: /scomma %destinationDirectory%\WebBrowserPassView.csv
+        ExportFormat: csv
+               
+# Documentation
+# WebBrowserPassView is a password recovery tool that reveals the passwords stored by the following Web browsers: Internet Explorer (Version 4.0 - 11.0), Mozilla Firefox (All Versions), Google Chrome, Safari, and Opera. This tool can be used to recover your lost/forgotten password of any Website, including popular Web sites, like Facebook, Yahoo, Google, and GMail, as long as the password is stored by your Web Browser.
+# **Do not download** from the following location httpx://www.nirsoft.net/toolsdownload/webbrowserpassview.zip
```

Comment:
What does this mean? Why have this here if you don't want people to download from this link? Especially since the BinaryUrl is this same link?

Reply:
I'm changing the BinaryUrl to `https://www.nirsoft.net/protected_downloads/passreccommandline.zip` because some people have scripts to download whatever is listed in the BinaryURL field, so we need the correct BinaryURL there